command/repo.go

```diff
@@ -366,7 +370,7 @@ func repoFork(cmd *cobra.Command, args []string) error {
 				return fmt.Errorf("did not understand argument: %w", err)
 			}
 		} else {
-			repoToFork = ghrepo.FromFullName(repoArg)
+			repoToFork, _ = ghrepo.FromFullName(repoArg)
```

Comment:
You should handle the error return value here, and then you can cleanup the next conditional because the blankness of values is now checked within FromFullName 🎉

Reply:
done 👍